apps/navstar_overlay/src/navstar_overlay_poller.erl

```diff
@@ -185,6 +194,14 @@ scheme() ->
             "http"
     end.
 
+update_poll_period(OldPollPeriod) ->
+    case (NewPollPeriod = OldPollPeriod * 2) > ?MAX_POLL_PERIOD of
```

Comment:
Please break up the initial part of the statement and the later part of the statement. Alternatively, use a guard on the update_poll_period function itself.